salt/backups/restore.sls

```diff
@@ -0,0 +1,30 @@
+install_duplicity:
+  pkg.installed:
+    - pkgs:
+        - duplicity
+        - python-pip
+
+install_duplicity_backend_requirements:
+  pip.installed:
+    - name: boto
+
+{% for service in salt.pillar.get('backups:enabled', []) %}
+{% if service.get('pkgs') %}
+install_packages_for_{{ service.title }}_backup:
+  pkg.installed:
+    - pkgs: {{ service.pkgs }}
```

Comment:
This would be a list of packages that need to be installed for the script to work. e.g. mysql-client, curl, etc.